### DISCUSSION TITLE
翻译引擎无法连接到网络，urllib.error.URLError: <urlopen error [WinError 10054] 远程主机强迫关闭了一个现有的连接。>

### DISCUSSION THREAD

**Discussion by FlameU4801** [bookfere/Ebook-Translator-Calibre-Plugin#180] [Q&A]
目前Google fan翻译引擎及ChatGPT均出现此问题
![capture_20231126200028475](https://github.com/bookfere/Ebook-Translator-Calibre-Plugin/assets/152068898/79fc7b80-d585-49a6-820c-292430869d3b)

代理已连接成功
![capture_20231126195530279](https://github.com/bookfere/Ebook-Translator-Calibre-Plugin/assets/152068898/f30f3f02-6844-476b-a309-c02aac2df96a)

**Comment by bookfere** [comment]
插件的代理测试功能目前只是简单判断插件和代理客户端之间是否正常通信，无法测试代理是否真正可用。

从你给出的错误信息看应该就是代理的问题，建议检查代理配置，或者换个代理试试看。
